src/Input/Input.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import classNames from "classnames";
+import styled from "styled-components";
+
+const InputContainer = styled.div`
+  display: flex;
+`;
+const PrefixIcon = styled.span`
```

Comment:
I think naming the icons `left` and `right` is better. `prefix` and `suffix` are for words